lib/tasks/remove-files.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// Creates a directory with the name directoryName in cwd and then sets cwd to
+// this directory.
+
+var Promise     = require('../ext/promise');
+var Task        = require('../models/task');
+var rimraf  = require('rimraf');
+var _ = require('lodash');
+var chalk = require('chalk');
```

Comment:
We should either line up all the `=` or none of them. I lean towards none, but I don't care as long as it's consistent.